dist.sh

```diff
@@ -60,3 +73,14 @@ done
 for f in $core_files; do
     cp $f $dist_dir/
 done
+
+case $platform in
+    windows)
+        rm -f $dist_dir/workcraft
+        7z a -r ${dist_dir}.zip $dist_dir >/dev/null
+        ;;
+    linux)
+        rm -f $dist_dir/workcraft.bat
+        tar -czf ${dist_dir}.tar.gz $dist_dir
```

Comment:
Unfortunately this will put the whole path `dist/workcraft-TAG-linux` inside `workcraft-TAG-linux.tar.gz`, not just the required `workcraft-TAG-linux` directory :( The same for windows-specific code.